pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -74,10 +75,29 @@ export class MobyClient implements ContainerEngineClient {
     // fails to do so).
     while (successCount < 10) {
       try {
-        await this.runClient(['system', 'info'], 'ignore');
+        await this.runClient(['system', 'info'], 'pipe');
         successCount++;
+        failureCount = 0;
       } catch (ex) {
         successCount = 0;
+        failureCount++;
+        if (failureCount > 10) {
+          // If we've been error for a while, log the output.
+          if (ex && typeof ex === 'object') {
+            const output = { stdout: '', stderr: '' };
+
+            if ('stdout' in ex && typeof ex.stdout === 'string') {
+              output.stdout = ex.stdout;
+            }
+            if ('stderr' in ex && typeof ex.stderr === 'string') {
+              output.stderr = ex.stderr;
+            }
+            if (output.stdout !== lastOutput.stdout || output.stderr !== lastOutput.stderr) {
+              console.error('Failed to run docker system info (will retry):', output);
```

Comment:
Maybe include the `failureCount` in the error message? It is probably obvious by how many times the error is repeated with no other messages in between, but it doesn't hurt to be verbose.